ui/pages/confirmations/hooks/useEIP7702Networks.ts

```diff
@@ -103,6 +103,17 @@ export const useEIP7702Networks = (address: string) => {
         }
       });
 
+      // Sort by chainIdHex to ensure consistent ordering for comparison
+      networksSupporting7702.sort((a, b) => {
+        if (a.chainIdHex < b.chainIdHex) {
+          return -1;
+        }
+        if (a.chainIdHex > b.chainIdHex) {
+          return 1;
+        }
+        return 0;
+      });
```

Comment:
What is the ultimate expectation here? If we are really want to do sort it decimal chainIds it's not entirely correct since `"0xa" < "0x10"` will be `true` but in fact it's wrong.

Could this be simplified to this? Also does it warrant a unit test?
 ```suggestion
      // Sort by chainIdHex to ensure consistent ordering for comparison
      networksSupporting7702.sort((a, b) => {
         const aChainIdNum = parseInt(a.chainIdHex, 16);
         const bChainIdNum = parseInt(b.chainIdHex, 16);
         return aChainIdNum - bChainIdNum;
      });
```